### PR TITLE
tispark:add daily

### PR DESCRIPTION
**Pull request by shiyuhang0**
tispark dialy ci

### CODE REVIEW

**Comment by shiyuhang0** [comment]
@purelind PTAL

File: tispark/daily.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+version: 1
+
```
Comment (purelind) [inline tispark/daily.yaml:2]: please add branch:
`defaultRef: master`

Comment (shiyuhang0) [reply tispark/daily.yaml:3]: done

File: tispark/daily.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      image: hub.pingcap.net/jenkins/centos7_golang-1.13_java:latest
+      request:
+        cpu: 2
+        mem: 4Gi
+      limit:
+        cpu: 2
+        mem: 4Gi
+
+# need change
+  - name: tispark_ghpr_common_test
```
Comment (purelind) [inline tispark/daily.yaml:46]: name need to change `daily-trigger-tispark-ingegration-test`

**Review by purelind — APPROVED** [review]

**Pull request merged**